clojure_action.py

```diff
@@ -265,14 +265,20 @@ def _setOutput(name:str, value:str):
     # parse the results
     licenses_errors = set()
     vulns = set()
+    license_vulns = set()
     try:
         for lic, _v in test_res['licensesPolicy']['orgLicenseRules'].items():
             licenses_errors.add(lic)
     except KeyError:
         logging.error(f"Error parsing licenses!")
     try:
         for vuln in test_res['vulnerabilities']:
-            vulns.add(VulnReport(vuln))
+            report=str(VulnReport(vuln))
+            licenseString='snyk:lic'
+            if(licenseString in report):
+                license_vulns.add(report)
+            else:
+                vulns.add(report)
```

Comment:
You're changing the behavior here. Vulns is being changed from a `VulnReport` to a `String`. We can accomplish the same thing with a check here like:

```
if vuln['id'].startswith('snyk:lic:')
```

Reply:
Changed the commented lines to: 
```
if(vulns['id'].startswith('snyk:lic:') is False):
```
If the license vulnerabilities need to be anywhere in the future I can add the license_vulns back but right now it did not have a purpose.